src/emulator/storage/apis/gcloud.ts

```diff
@@ -151,7 +151,7 @@ export function createCloudEndpoints(emulator: StorageEmulator): Router {
       }
       throw err;
     }
-    return res.send(200).json(new CloudStorageObjectMetadata(metadata));
```

Comment:
Not sure why you made this change, but `status(200)` should work?

Reply:
Interestingly enough, this was causing integration tests to fail with this error:

```
[1 stderr] (node:1087696) UnhandledPromiseRejectionWarning: Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers 
after they are sent to the client                                                                             
    at ServerResponse.setHeader (_http_outgoing.js:530:11)                                                    
    at ServerResponse.header (/usr/local/google/home/tonyjhuang/dev/firebase-tools/node_modules/express/lib/re
sponse.js:771:10)                                                                                             
    at ServerResponse.send (/usr/local/google/home/tonyjhuang/dev/firebase-tools/node_modules/express/lib/resp
onse.js:170:12)                                                                                               
    at ServerResponse.json (/usr/local/google/home/tonyjhuang/dev/firebase-tools/node_modules/express/lib/resp
onse.js:267:15)                                                                                               
    at /usr/local/google/home/tonyjhuang/dev/firebase-tools/lib/emulator/storage/apis/gcloud.js:115:30        
    at processTicksAndRejections (internal/process/task_queues.js:97:5)                                       
(node:1087696) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by 
throwing inside of an async function without a catch block, or by rejecting a promise which was not handled wi
th .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejec
tions=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)          
(node:1087696) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promi
se rejections that are not handled will terminate the Node.js process with a non-zero exit code. 
```